src/main/java/songbox/house/task/VkDownloadTrackTask.java

```diff
@@ -5,6 +5,7 @@
 import com.mpatric.mp3agic.Mp3File;
 import lombok.extern.slf4j.Slf4j;
 import songbox.house.domain.entity.VkAudio;
+import songbox.house.util.ThreadChange;
```

Comment:
is it used?

Reply:
This is just an **UNUSED** import, and yes it not used

Reply:
unused imports clog the code and are useless